src/main/java/UnivusException.java

```diff
@@ -0,0 +1,5 @@
+public class UnivusException extends Throwable{
```

Comment:
Throwable should be followed by a whitespace
```suggestion
public class UnivusException extends Throwable {
```